sdk/tests/deqp/modules/shared/glsFboCompletenessTests.js

```diff
@@ -927,7 +927,7 @@ goog.scope(function() {
                 throw new Error('Multiple attachments not supported');
             var maxAttachments = gl.getParameter(gl.MAX_COLOR_ATTACHMENTS);
 
-            for (var i = 0; i < maxAttachments; ++i) {
+            for (var i = 1; i < maxAttachments; ++i) {
                 this.attachTargetToNew(gl.COLOR_ATTACHMENT0 + i, this.m_params.colornKind,
```

Comment:
The failed cases are none_rbo_none_none and none_tex_none_none. For these two test cases, parameter this.m_params.colornKind would be RENDERBUFFER / TEXTURE, but this.m_params.color0Kind (called by another attachTargetToNew on line 922#) would be 0, it will not attach any image to COLOR_ATTACHMENT0 at line 922#. So if the iteration begins from 1, instead of 0, the COLOR_ATTACHEMNT0 would have no image to attach. 

But readBuffer's target is COLOR_ATTACHMENT0 by default, So it fails on MacOS, because readBuffer's target has no image attached. 

If you simply add one line code: gl.readBuffer(gl.COLOR_ATTACHMENT1), it can pass.